README.md

```diff
@@ -68,3 +68,9 @@ ArgoCD should successfully get Nautobot deployed. Now come the OpenStack
 components which aren't working with GitOps methods at this time.
 
 [Install Keystone](./components/10-keystone/README.md)
+
+### Setup Argo Workflow
+
+```bash
+kubectl kustomize --enable-helm components/11-argo-workflow/ | kubectl apply --server-side -f -
```

Comment:
So here you say it needs server side apply but then the ArgoCD config isn't using server side apply. Which is it?

Reply:
You also shouldn't need to add this to the top level readme since Argo should deploy this automatically.

Reply:
I was wondering that. OK I'll remove this